docs/howto/platform-sdk-quickstart-guide.md

```diff
@@ -0,0 +1,196 @@
+-   Alteryx Designer Version 2021.2\*
+-   Python Version 3.8.5
+-   [pip](https://pypi.org/) (automatically installed with Python 3.8.5)
+-   [node](https://nodejs.org/en/download/) [14](https://nodejs.org/en/blog/release/v14.17.3/)
+-   [Git](https://git-scm.com/downloads)
+
+\*Alteryx Designer version 2021.4 is required to use the Alteryx Python
+SDK v2.0.0.
+
+### CLI
+
+The AYX Plugin CLI provides you with a set of utilities to manage your
+plugin and is a great starting point for building a tool. AYX Python SDK
+installation is also covered in this section.
+
+Follow these steps to get started with the CLI:
+
+1.  Run this command to install the AYX Plugin CLI:\
+    \
+    `pip install ayx-plugin-cli`\
+     
+2.  To verify that you installed the CLI properly, run this command:\
+    \
+    `ayx_plugin_cli version`\
+    \
+    If the installation was successful the above command returns the
+    version number. If the CLI was not installed properly, the
+    `ayx_plugin_cli` command is not recognized in the terminal, and your
+    terminal reflects that.\
+     
+3.  Run this command to install the AYX Python SDK:\
+    \
+    `pip install ayx-python-sdk`\
+    \
+    *(Optional) Create a new directory to house all of your workspaces
+    and plugins, and continue the steps in this guide from the new
+    directory. This helps keep your folder structure clean.*\
+     
+4.  Once the installation is successful, run this command to create a
+    workspace:\
+    \
+    `ayx_plugin_cli sdk-workspace-init`\
+    \
+    The command asks you to provide information about your workspace
+    through a series of prompts.\
+     
+5.  Once you have stepped through the above command, run this command to
+    create a custom tool or plugin:\
+    \
+    `ayx_plugin_cli create-ayx-plugin`\
+    \
+    The command asks you to provide information about your tool or
+    plugin through a series of prompts. Note that this operation can
+    take a minute to complete.
+
+Once you have stepped through the above command, you can go to the
+location of your workspace and view the files that were created for the
+new plugin.
+
+For more information about the CLI, please visit the [reference
+documents](https://help.alteryx.com/developer-help/ayx-plugin-cli-overview)
+
+### UI SDK
+
+The UI SDK consists of 2 pieces.
+
+-   The 1st piece is a React-based [component
+    library](https://alteryx.github.io/alteryx-ui/)
+    that enables you to build the interface for your tool or plugin.
+-   The 2nd piece is a [communication
+    bridge](https://alteryx.github.io/react-comms/)
+    that lets you interact with Alteryx Designer and persists the values
+    that you want to save between runs of a workflow, clicking on and
+    off a tool. These are also the values that are available to the
+    engine (back end) of your tool.
+
+For the real-time Dev Harness that assists in developing a tool UI,
+check
+out [this](https://github.com/alteryx/dev-harness)
```

Comment:
TW review:

For the real-time Dev Harness that assists in tool UI development, refer to [Dev Harness](https://github.com/alteryx/dev-harness).